.github/workflows/cla.yml

```
@@ -7,7 +7,9 @@ on:
 
 jobs:
   CLABot:
-    if: github.event_name == 'pull_request_target' || contains(github.event.comment.html_url, '/pull/')
+    if: |
+      (github.event_name == 'pull_request_target' || contains(github.event.comment.html_url, '/pull/')) &&
+      (github.ref != 'refs/heads/l10n_crowdin_action' && github.event.issue.user.login != 'github-actions')
```

I think this should be a OR.

```suggestion
      (github.ref != 'refs/heads/l10n_crowdin_action' || github.event.issue.user.login != 'github-actions')
```

Or we could switch this to a negation, making it easier to read:

```suggestion
      !(github.ref == 'refs/heads/l10n_crowdin_action' && github.event.issue.user.login == 'github-actions')
```

---

Right now this would skip the CLA bot for any branches with that name and any PRs created by the GitHub Actions user. So forks would still skip this.